### PR TITLE
Pull images from Firebase Storage for RequestGroup Form Modal

### PR DESCRIPTION
**Pull request by chamod-gamage**
+ Created function in `services/storage.ts` to read from Firebase Storage folder given the folder's name and return an array of URLs
+ Added `useEffect` in RequestGroupForm modal to call that function and setImages to the result
+ Added a length > 1 conditional on rendering ImageList and added a transition to make things smoother since the image call takes a second longer
+ Uploaded photos specified in this [doc](https://docs.google.com/document/d/1FwUkUiXpC61jjYmZ_hREan0zTEZ8h835txZifoZg9As/edit?ts=6060da83) to firebase storage

[Demo](https://drive.google.com/file/d/170UawK1sI6OOXeKjCtH3oQnM4wSWYY-K/view?usp=sharing)

### CODE REVIEW

File: client/src/components/organisms/RequestGroupForm.tsx [hunk 1]
```diff
@@ -1,147 +1,160 @@
...
+  const [descriptionError, setDescriptionError] = useState("");
+  const [imageError, setImageError] = useState("");
+  const [requestTypesError, setRequestTypesError] = useState("");
+
+  useEffect(() => {
+    async function getImages() {
+      setImages(await getFilesFromFolder(imageFolder));
+    }
+    getImages();
+  }, []);
```
Comment (chamod-gamage) [inline client/src/components/organisms/RequestGroupForm.tsx:58]: ignore any diffs in this file below this line - was just my formatter

**Comment by bonnie-chin** [comment]
lgtm!! the jump is obvs not ideal but I trust y'all to make the best call on what you have time to take care of -- also I did not look at the code LOL so don't take my approval too literally hehe

**Review by bonnie-chin — APPROVED** [review]

**Review by meganniu — APPROVED** [review]
LGTM!!! thanks @chamod-gamage for taking ownership of this <3333

**Comment by chamod-gamage** [comment]
Thx for the quick reviews~

**Pull request merged**